<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projetSparebox</class>
 <widget class="QMainWindow" name="projetSparebox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>projetSparebox</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="GroupBoxCreationResponsable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>291</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Création d'Utilisateur</string>
    </property>
    <widget class="QPushButton" name="pushButtonCreerResponsable">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Créer</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelPrenom">
        <property name="text">
         <string>Prenom: </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Prenom">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="labelNumTel">
        <property name="text">
         <string>Numero Telephone: </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_NumTel">
        <property name="maxLength">
         <number>10</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QRadioButton" name="radioButtonTechnicien">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>320</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Technicien</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonResponsables">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Responsables</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="labelMail">
        <property name="text">
         <string>Adresse Mail :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Mail">
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="maxLength">
         <number>255</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="LabelNom">
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Nom">
        <property name="maxLength">
         <number>50</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>216</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mot de passe utilisateur :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mdp">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText</set>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonQuitter">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>450</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>395</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>projetSparebox</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
